core/node/crypto/encrypt.go

```diff
@@ -0,0 +1,5 @@
+package crypto
+
+const (
+	AES_GCM_DERIVED_ALGORITHM = "r.aes-256-gcm.derived"
+)
```

Comment:
can you delete this file and this constant? the stream nodes don't care about what encryption you're using (yet). It's currently only for testing, please just send the algorithm string from the test